README.md

```diff
@@ -39,9 +39,28 @@ GOV.UK Frontend supports:
 
 - [recommended browsers](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in)
 - [recommended assistive technologies](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#which-assistive-technologies-to-test-with)
-- Internet Explorer 8, 9 and 10, although components may not look perfect
 - your users overriding colours in Windows, Firefox and Chrome
 
+### Internet Explorer (IE)
+
+We're planning to [reduce our level of support for all versions of IE](https://github.com/alphagov/govuk-frontend/discussions/2607).
+
+In a future major release, we'll reduce the level of support for IE11 from 'compliant' to 'functional'.
```

Comment:
Should we add some detail here about what 'functional' and 'compliant' mean, or would that just be repeating what's in Kim's proposal (which we're linking to)?